components/projects/NewProject/NewProjectForm.tsx

```diff
@@ -121,18 +121,23 @@ function NewProjectForm({ employees, customers }: NewProjectFormProps): JSX.Elem
                         }
                     ></Input>
                 </FormControl>
-                <FormControl isRequired={true}>
-                    <FormLabel>Customer</FormLabel>
-                    <Select onChange={handleCustomerChange} placeholder="Select customer">
-                        {customers?.map((el, idx) => {
-                            return (
-                                <option key={idx} value={el.id}>
-                                    {el.name}
-                                </option>
-                            );
-                        })}
-                    </Select>
-                </FormControl>
+                <Flex flexDirection="row" justifyContent="center">
+                    <FormControl isRequired={true}>
+                        <FormLabel>Customer</FormLabel>
+                        <Flex flexDirection="row" justifyContent="space-between">
+                            <Select onChange={handleCustomerChange} placeholder="Select customer" marginRight="0.3rem">
+                                {customers?.map((el, idx) => {
```

Comment:
I think you could use more descriptive variable naming, unless it's a preference to omit it? It's an array of customers so you could use 
```typescript
customers?.map((customer, idx) => {
```
etc

Reply:
el changed to customer in 9137767